test/puppetlabs/puppetdb/http/metrics_test.clj

```diff
@@ -37,22 +33,23 @@
   [req]
   (assoc-in req [:headers "accept"] "text/plain"))
 
-(deftestseq metrics-set-handler
+(let [app (server/build-app)]
+  (deftestseq metrics-set-handler
```

Comment:
Okay, I get it. I think the way we're handling it more commonly is with the with-http-app fixture. That seems a bit cleaner to me but might take some modification to get working with the new metrics prefix. I haven't looked too much into it but maybe it'd be a worthwhile update.